src/index.html

```diff
@@ -1,12 +1,377 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="moyo-header">
+
+      <a href="#home" class="moyo-header__logo">
+        <img src="./images/logo.png" alt="Moyo logo">
+      </a>
+
+      <nav class="moyo-header__nav">
+        <a class="moyo-header__nav-link moyo-header__nav-link--is-active is-active" href="http://">Apple</a>
+        <a class="moyo-header__nav-link" href="http://">Samsung</a>
+        <a class="moyo-header__nav-link" href="http://">Smartphones</a>
+        <a
+          class="moyo-header__nav-link"
+          data-qa="nav-hover"
+          href="http://"
+        >
+          Laptops & Computers
+        </a>
+        <a class="moyo-header__nav-link" href="http://">Gadgets</a>
+        <a class="moyo-header__nav-link" href="http://">Tablets</a>
+        <a class="moyo-header__nav-link" href="http://">Photo</a>
+        <a class="moyo-header__nav-link moyo-header__nav-link--last-link" href="http://">Video</a>
+      </nav>
+    </header>
+
+    <main class="moyo-catalog">
+      <div class="products">
+        <div class="products__product-wrap">
+          <div class="card" data-qa="card">
+            <img
+              class="card__image"
+              src="./images/imac.jpeg"
+              alt="TV APPLE A1419 image"
+            >
+            <div class="card__name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </div>
+            <div class="card__code">
+              Product code: 195434
+            </div>
+
+            <div class="card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="card__reviews">
+                Reviews: 5
+              </div>
+            </div>
+
+            <div class="card__price-block">
+              <div class="card__price-text">
+                Price:
+              </div>
+              <div class="card__price-value">
+                $2,199
+              </div>
+            </div>
+
+            <a
+              href="//"
+              class="card__buy-link card__buy-link--hover"
+              data-qa="card-hover"
+            >
+              Buy
+            </a>
+          </div>
+        </div>
+
+        <div class="products__product-wrap">
+          <div class="card">
```

Comment:
To not create an extra div wrapper you can combine them together into one div with two classes

```suggestion
        <div class="card products__product-wrap">
```